README.md

```diff
@@ -444,15 +438,17 @@ The plugin is currently used on an opt-in basis. To use it, apply the plugin and
 apply plugin: 'com.palantir.baseline-java-versions'
 
 javaVersions {
+    compiler = 25
     libraryTarget = 11
     distributionTarget = 17
     runtime = 21
 }
 ```
 
 The configurable fields of the `javaVersions` extension are:
-* `libraryTarget`: (required) The Java version used for compilation of libraries that are published.
-* `distributionTarget`: (optional) The Java version used for compilation of code used within distributions, but not published externally. Defaults to the `libraryTarget` version.
+* `compiler`: (required) The version of the Java compiler used. 
```

Comment:
I have made this required because if it is not, everything remains unset and Gradle defaults to using the daemon JDK version, which seems not ideal. But it does mean it always needs to be set, even if your project only has groovy/scala and no java.